lib/multi_blas_core.cuh

```diff
@@ -242,8 +243,8 @@ void multiblasCuda(const Complex *a, const double2 &b, const double2 &c,
       ypp.push_back(&yp[i]); wpp.push_back(&wp[i]);
     }
 
-    multiblasCuda<NXZ, NYW, RegType, StoreType, yType, M, Functor, writeX, writeY, writeZ, writeW>
-       (a, b, c, xpp, ypp, zpp, wpp, length);
+    multiblasCuda<NXZ, RegType, StoreType, yType, M, Functor, writeX, writeY, writeZ, writeW>
+      (a, b, c, xpp, ypp, zpp, wpp, NYW, length);
```

Comment:
This breaks compilation with clang: 
`error: variable length array of non-POD element type 'Spinor<float4, short4, 6, 1, 1>`